### ISSUE TITLE
release candidate 0.1.0

### ISSUE THREAD

**Issue by jwerle** [sphia/sphia#30]
- [ ] Unit tests
- [ ] No leaks(#32)
- [x] File based configuration(#15)

**Comment by jwerle** [comment]
Where do we want to be for a release candidate?

**Comment by stephenmathieson** [comment]
imo, before 0.1.0 we should minimally have:
- a full set of unit tests (to verify we don't break shit in x.x.x)
- less leaks (though i think this is being worked on by @larzconwell)

also, i think we aught to put #23, #24 and #25 on hold util ~0.1.0.

**Comment by larzconwell** [comment]
Yeah that sounds good to me @stephenmathieson, I'd also like to get #15 in but that could be something lower priority.

**Comment by jwerle** [comment]
Yep yep yep yep and YEP!!

**Comment by jwerle** [comment]
And...... Yep!
